blackbox/docs/administration/snapshots.txt

```diff
@@ -1,8 +1,9 @@
 .. highlight:: psql
 .. _snapshot-restore:
 
-Backup With Snapshot and Restore
-================================
+=========
+Snapshots
```

Comment:
I would prefer to keep it like `Backup and Restore With Snapshots`
This way it would be clear that this is the place the users can read on how to backup/restore their DB.

Reply:
Don't have a strong opinion there, so we can go with `snapshots`.
As discussed maybe we refer to `backup` in the guide.

Reply:
@matriv per our slack discussion. I wanna keep this as "snapshots" for now. and let's maybe think about adding a "backup" doc to the guide later. thx!